cEP-0004.md

```diff
@@ -95,5 +113,14 @@ of them.
 Usage of letters would make usage of coala faster, as one could just remember
 that `a` stands for 'apply patch' for instance, and they would not have to check
 the list for every result to see what number 'apply patch' has been assigned.
-This change would also drastically reduce the required minimum amount of
-characters needed for th
+
+
+### Allow chaining actions in a single input
```

Comment:
I was thinking about generating multiple patches. Let's say that the user have multiple sections in a coafile. Many of them contains BearA, coala will generate a special patch named patch BearA. If the user apply `patch BearA`, BearA will affect all the files from sections where it was supposed to run. We could give user a `preview` of what changes will apply a special patch.